src/components/Search/SelectedRevisionsTable/SelectedRevisionsTable.tsx

```diff
@@ -24,14 +24,13 @@ const tableHeaderDetails: TableHeaderDetails[] = [
   { id: 5, title: 'Timestamp', alignment: 'left' },
 ];
 
-function SelectedRevisionsTable(props: SelectedRevisionsProps) {
+function SelectedRevisionsTable(
+  props: SelectedRevisionsProps,
+): JSX.Element | null {
```

Comment:
once you've changed the render below, I don't think you'll need a type for return value anymore